src/contentful/ContentfulClient.ts

```diff
@@ -15,8 +15,8 @@ import {
 
 export class ContentfulClient {
 
-    private static readonly PREVIEW_URL = "https://preview.contentful.com";
-    private static readonly PRODUCTION_URL = "https://cdn.contentful.com";
+    public static readonly PREVIEW_URL = "https://preview.contentful.com";
```

Comment:
do these need to be public?

Reply:
I made it public, so that it could be accessible from the unit test. I think the argument could be made to use hard coded expected values in the unit tests instead.